src/App.test.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
+import { shallow } from 'enzyme';
+
 import App from './App';
 
 it('renders without crashing', () => {
-  const div = document.createElement('div');
-  ReactDOM.render(<App />, div);
-  ReactDOM.unmountComponentAtNode(div);
+  const root = shallow(<App />);
```

Comment:
I'm curious, what happens with the IO that should be fired here?

Reply:
This actually only renders the main level so we don't have to worry about that as part of this. We should solve problems related to that in a follow-up issue

Reply:
Ah gotcha, nevermind :)